app/models/form.rb

```diff
@@ -7,21 +7,22 @@ class Form
 
   def initialize(form_path, name, sections_in_form = [], type = "lettings")
     if type == "sales"
-      # @form_definition = sections.map do |s|
-      #   s.new(nil, nil, self)
-      # end
       @name = name
-      # @setup_sections = [Form::Setup::Sections::Setup.new(nil, nil, self)]
-      # @form_definition = JSON.parse(File.open(form_path).read)
+      @setup_sections = [Form::Sales::Setup::Sections::Setup.new(nil, nil, self)]
       @form_sections = sections_in_form.map { |sec| sec.new(nil, nil, self) }
       @type = "sales"
-      @sections = form_sections
+      @sections = setup_sections + form_sections
       @subsections = sections.flat_map(&:subsections)
       @pages = subsections.flat_map(&:pages)
       @questions = pages.flat_map(&:questions)
       @start_date = Time.zone.local(name[0..3], 4, 1)
```

Comment:
IMO we should replace `name` with the start year if that's how we would like to use it.

Reply:
So that's a good point, I think the intention was to make it work the easiest way possible to unblock the rest of the sales tickets.
There was a conversation about structuring our the list of forms in a little bit different way within the form handler, something like this: [{type: "sales", start_year: "2022" }, {type: "lettings", start_year: "2021" }...] etc.
I think maybe not as part of this PR, but immediately after as part of this piece of work

Reply:
Current, next and previous might be other good labels that don't need updating every year